### PR TITLE
Create comment mutation

### PR DESCRIPTION
**Pull request by 7alip**
If you think it's working properly, we need to activate "comment approval process" in the backend.

Currently your comment will not be shown immediately after you send it, unless you refresh the page or refocus on the page. This is not necessary as we want to list only approved comments. Feel free to add your opinion on this feature.

### CODE REVIEW

**Comment by 7alip** [comment]
I tried to send a comment as `Cok guzel bir fotograf. Cok anlamli olmus`. I couldn't send it because of error. When I check the error message from Network tab, I see this message: `Bad language used! Please polite your comment...`.

Maybe we can disable `Bad word filtering` option as we have already approval process. Otherwise people won't be able to understand why sending comment is failing.

Another solution could be to warn the users about changing their message. But this sounds pretty weird. Because they would be warned for a normal message just like I got for  this, `Cok guzel bir fotograf. Cok anlamli olmus`.

What's your idea @MustafaTRKYLMZ @ertugrul-aydin?

**Comment by MustafaTRKYLMZ** [comment]
> I tried to send a comment as `Cok guzel bir fotograf. Cok anlamli olmus`. I couldn't send it because of error. When I check the error message from Network tab, I see this message: `Bad language used! Please polite your comment...`. (But I could send `Çok güzel bir fotoğraf. Çok anlamlı olmuş` with Turkish characters).
> 
> Maybe we can disable `Bad word filtering` option as we have already approval process. Otherwise people won't be able to understand why sending comment is failing.
> 
> Another solution could be to warn the users about changing their message. But this sounds pretty weird. Because they would be warned for a normal message just like I got for this, `Cok guzel bir fotograf. Cok anlamli olmus`.
> 
> What's your idea @MustafaTRKYLMZ @ertugrul-aydin?

I think we don't need to use Bad word filtering because of the controling. When we approving the comment we still will check it. That's why we don't need "bad word filtering"

File: src/pages/club/art/[slug].js [hunk 1]
```diff
@@ -40,7 +40,7 @@ const ArtPage = ({ seo }) => {
               {/* Single Art Comments */}
               <Stack spacing={4}>
                 {/*  Comment form */}
-                <CommentForm user={user} />
+                <CommentForm auth={auth} artId={artQuery.data?.id} />
 
                 {/*List comments of the current art */}
```
Comment (MustafaTRKYLMZ) [inline src/pages/club/art/[slug].js:45]: I think we should put scroll here for more comments.

**Review by MustafaTRKYLMZ — APPROVED** [review]
Thank you for your afford

**Comment by 7alip** [comment]
> In my opinion, Everything looks good.But why have to we  refresh the page for we can see the comment we created ?

As I mentioned, I didn't implement optimistic ui or invalidate query approach (to immediately show the comment) after sending a comment. Because in production, users will never see the comment immediately. The comment will be shown after it's approved by an editor.

**Pull request merged**